concent_api/core/subtask_helpers.py

```diff
@@ -258,3 +269,39 @@ def get_one_or_none(
         instances = subtask_or_query_set.filter(**conditions)
         assert len(instances) <= 1
         return None if len(instances) == 0 else instances[0]
+
+
+def finalize_deposit_claim(
+    subtask_id: str,
+    concent_use_case: ConcentUseCase,
+    ethereum_address: str,
+    public_key: str,
+) -> None:
+    deposit_claim = get_one_or_none(
```

Comment:
When `get_one_or_none` was created @cameel wanted it to be generic, not only for subtasks. It was changed in the meantime (it was used only for Subtasks) but it can be used also for others Models. Fixing this function is not this pull request, if you can do it to be generic again please do. But here at least please use:
`deposit_claim: DepositClaim = get_one_or_none(`

Reply:
Wasn't the issue that `get_or_none()` has `Subtask` as its argument type or something like that? I remember it from another review.

Reply:
Arguments are fine, the returned type was `Optional[Subtask]`, I have changed it to be more generic.